### DISCUSSION TITLE
how to highlight problematic/compile failed files?

### DISCUSSION THREAD

**Discussion by peczenyj** [zed-industries/zed#15002] [Feature Requests]
Hello

I am a golang dev trying to use Zed.

Using vscode/atom/eclipse I can cleary see what are the files with some compile issue because there is a highlight on the file tree (usually in red), and it is easy to open and find the problematic lines.

It seems the highlight on source code problems seems really minimalistic and I need to open the file to find it, usually after run a make test or something that points what is the file with a problem

Question: is it an issue on my theme (dark), with golang support or perhaps I need to configure it? Because for me this is a very interesting feature, specialty when I need to manage old repositories.

Will be really good immediatelly find the files with any problem, then cleary identify the lines

Thanks

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
